src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportHistory/__tests__/ErrorReportHistoryViewModelTests.swift

```diff
@@ -0,0 +1,44 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class ErrorReportHistoryViewModelTests: XCTestCase {
+
+	let model = ErrorReportHistoryViewModel()
+	
+	func testSectionsOfViewModel() throws {
+		// GIVEN
+		let tableViewModel = model.dynamicTableViewModel
+		
+		// WHEN
+		let numberOfSections = tableViewModel.content.count
+		
+		// THEN
+		XCTAssertEqual(numberOfSections, 2)
+	}
+	
+	func testForNumberOfStaticCells() throws {
+		// GIVEN
+		let tableViewModel = model.dynamicTableViewModel
+
+		// WHEN
+		let numberOfCells = tableViewModel.section(0).cells.count
+
+		// THEN
+		XCTAssertEqual(numberOfCells, 2)
```

Comment:
I would expect 5 cells for 5 (mocked) IDs. Why 2?

Reply:
This test if for static cells as the test name suggest which are only two.

Reply:
Ok got it. Could you add a `TODO` into this test to extend it for the number of IDs and displayed cells. Once the model gets some data injected we can and should test this as well.

Reply:
I will add it :)